src/ray/gcs/gcs_client/gcs_client.h

```diff
@@ -184,6 +184,31 @@ class RAY_EXPORT GcsClient : public std::enable_shared_from_this<GcsClient> {
   std::function<void()> resubscribe_func_;
 };
 
+class RAY_EXPORT GcsSyncClient {
```

Comment:
maybe add comments this is only supposed to use by cython?

Reply:
done :)

Reply:
Should we then name it more explicitly?